packit/api.py

```diff
@@ -1523,7 +1528,15 @@ def push_and_create_pr(
                 target_branch=git_branch,
             )
         else:
-            logger.debug(f"PR already exists: {pr.url}")
+            logger.debug(
+                f"PR already exists: {pr.url} updating title"
+                f' "{pr_title}" and description "{pr_description}"',
```

Comment:
```suggestion
                f"PR already exists: {pr.url}, updating title with"
                f' "{pr_title}" and description with "{pr_description}"',
```

But I'm not sure if it's the best idea to log the entire description, as it is quite long.